src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -844,14 +844,20 @@ public Stream GetOutputStream(Stream fileStream)
             if (_isBackupEncrypted == false)
                 return fileStream;
 
+            byte[] encryptionKey;
```

Comment:
The function name is misleading.
This is getting the stream for the snapshot and here we always use the `MasterKey` of the database.

Reply:
I haven't introduced or modified this method. I can rename it if you want. Just give me the proper name for it.

Reply:
Sorry, I looked at the method in `DocumentDatabase`.
This looks fine.